lib/game_controller.rb

```diff
@@ -26,8 +26,7 @@ def start_game
     # funny enough we could actually make this work with a second player if we just
     # made their piece an O or something :)
     turn_counter = 1
-    while !board.board_is_full?
-
+    while !board.board_is_full? && !player.winner && !computer.winner
```

Comment:
if player.winner = false and computer.winner = false is this while loop technically saying while board is not full, player.winner = true and computer.winner = true, do the loop? I mean you demonstrated it works, but the logic doesn't make sense to me here for some reason.